book/scheduling.md

```diff
@@ -1503,7 +1505,8 @@ class Tab:
         self.js.interp.evaljs("__runRAFHandlers()")
         self.render()
         commit_data = CommitData(
-            self.url, self.scroll, document_height, self.display_list)
+            self.url, self.scroll, document_height, \
+            self.display_list)
```

Comment:
Does this matter? Does it go too long?

Reply:
In either case feel free to merge.

Reply:
Yeah it's because it caused horizontal scrollbars on the website.